subprojects/logging/src/main/java/org/gradle/api/logging/configuration/LoggingConfiguration.java

```diff
@@ -34,6 +34,11 @@
     void setLogLevel(LogLevel logLevel);
 
     /**
+     * Returns true if the log level has been explicitly set (as opposed to simply relying on the default value)
+     */
+    boolean isLogLevelSet();
```

Comment:
This is a chance of a public API and needs to indicate the version that changed it. Also should add `@Incubating`.